src/uct/api/uct.h

```diff
@@ -221,6 +234,10 @@ struct uct_iface_attr {
 
     size_t                   iface_addr_len; /**< Size of interface address */
     size_t                   ep_addr_len;    /**< Size of endpoint address */
+
+    double                   overhead;     /**< Message overhead, seconds */
```

Comment:
Please add more documentation for the parameter describing the goal, and the fact that it can be architecture depended.